README.md

```diff
@@ -1 +1,66 @@
-# -hello-microvers
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microvers project
+
+> This project  creating Github repository, branches and github flows
+> set up a new repository and test linter create two sections of the mobile website using the template 
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Please kindly follow the instruction given to you by your previouse reviewer this will help and guild you alot,Just as the reviewer has said here, kindly add a readme that looks like that of microverse while adhering strictly to the compulsory sections. You shouldn't copy the content by highlighting it but by clicking on the readme file in the repo and when it opens the readme content you should click on copy raw content [microverse readme look like this](https://user-images.githubusercontent.com/41163571/244160651-ea4f2102-9f00-45ea-855d-1ac0ef8a05b5.jpg)


- [ ] please use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) but remember to customize it to your project
- [ ] follow this step by step to customise your readme according to requirement README file should be descriptive:

All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in a student's repo. (if any relevant header is missing, kindly mark the PR as invalid)

The only optional section is FAQ.
Students should delete the live demo section if the student decides not to deploy the application. If project requirements say students should deploy, they should put the link.
Make sure the student has added the section Future Features, including 1-3 potential features to be added.


Students should remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.
It is the student's choice regarding how much into the details they go in each section of the README, you can advise but the student does not have to follow your suggestions.
Students should remove template logo. Optionally, the students can replace with their own logo if they have one.i hope this might help you .make sure you dont have a broken link especially your Mitlicense .